### PR TITLE
Add support for Apollo Federation

### PR DESCRIPTION
**Pull request by jinhong-**
As title says, this will add the --apollo-federation flag in cli and allow for apollo federation directives

### CODE REVIEW

File: website/docs/essentials/similar.md [hunk 1]
```diff
@@ -24,6 +24,7 @@ Run the following command:
 - `-r, --require <s>` - require a module
 - `-t, --token <s>` - an access token
 - `-h, --header <s>` - set http header (`--header 'Auth: Basic 123')
+- `--apollo-federation` - Support Apollo Federation directives (default: _false_)
```
Comment (kamilkisiela) [inline website/docs/essentials/similar.md:27]:
```suggestion
- `--federation` - Support Apollo Federation directives (default: _false_)
```

Maybe `--federation`?

**Review by kamilkisiela — COMMENTED** [review]
Nice and thank you :) I would love to see some tests but I can add it on my own

File: website/docs/essentials/similar.md (hunk 1 above)
Comment (jinhong-) [reply website/docs/essentials/similar.md]: @kamilkisiela i defer to writing the tests to you for this. I am not very sure what the testing conventions are and there were no existing tests inside @graphql-inspector/loaders for me to extend

**Comment by msnookxom** [comment]
I support these updates!  When do we expect them to be incorporated into the next `graphql-inspector` release?

**Pull request merged**